verification/verification.py

```
@@ -385,8 +391,7 @@ async def kick(self, int: discord.Interaction, member: discord.Member, reason: s
             user = await self.bot.fetch_user(member.id)
             await user.send(embed=embed)
         except Exception as e:
-            print(str(e))
-
+            logger.exception(f"Failed to send message to user {member.id}")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)